webbpsf/utils.py

```diff
@@ -676,19 +676,21 @@ def combine_docstrings(cls):
     return cls
 
 
-def to_griddedpsfmodel(HDUlist_or_filename=None, ext=0):
+def to_griddedpsfmodel(HDUlist_or_filename=None, ext_data=0, ext_header=0):
     """
     Create a photutils GriddedPSFModel object from either a FITS file or
     an HDUlist object. The input must have header keywords "DET_YX{}" and
-    "OVERSAMP" (will be present if psf_grid() is used to create the
-    file).
+    "OVERSAMP" (will already be present if psf_grid() is used to create
+    the file).
 
     Parameters
     ----------
-    HDUlist_or_filename : string
+    HDUlist_or_filename : HDUList or str
         Either a fits.HDUList object or a filename of a FITS file on disk
-    ext : int
-        Extension in that FITS file
+    ext_data : int
+        Extension of the data in the FITS file
+    ext_header : int
```

Comment:
What's the use case for pulling the data and header from different extensions?

Reply:
This was just in case someone had their own data (e.g. a single PSF in a fits file like from WebbPSF) that they wanted to turn into a `GriddedPSFModel` object, with the header information in the primary extension and the data in the 1st extension, which is a fairly common practice. I think this was requested by Lauren/MIRAGE, although I can't remember if it was for a case that still exists, or if it was for a patch of some kind.

Reply:
OK, I can't see immediately what that case might be, but that's fine in any case.